apps/fluent-tester/src/E2E/BootPage/specs/BootTestPage.spec.uwp.ts

```diff
@@ -18,68 +18,102 @@ const PAGE_TIMEOUT = 15000;
 describe('Open the app', function () {
```

Comment:
remind me, .win vs .win32 vs .uwp -- I thought we used .win for xaml/windows/rnw/uwp; what's UWP for, or what's the more correct understanding of .win?